src/libraries/System.IO.FileSystem/tests/Enumeration/AttributeTests.cs

```diff
@@ -113,18 +113,42 @@ public void DirectoryAttributesAreExpected()
         }
 
         [Fact]
-        public void IsHiddenAttribute()
+        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.OSX)]
+        public void IsHiddenAttribute_Windows_OSX()
         {
+            // Put a period in front to make it hidden on Unix
```

Comment:
this comment seems incorrect?
also nit, couple extra blank lines below

Reply:
Oops, the comments were inverted. Thanks.